docs/img/src/architecture.puml

```diff
@@ -0,0 +1,39 @@
+@startuml
+skinparam componentStyle rectangle
+
+package "OpenTelementry connector" {
+  [Processing data]
+  package "DataSetExporter" {
+    [Converts OTel structure into events] as convTraces
```

Comment:
Maybe be good to some how clarify (since there are many different type of "events" at play here) that it converts into into dataset-go event structure which also represents addEvents Event.